src/bench/bench.cpp

```diff
@@ -69,6 +69,9 @@ void benchmark::BenchRunner::RunAll(const Args& args)
         }
 
         Bench bench;
+        if (args.one_iteration) {
+            bench.epochs(1).epochIterations(1);
+        }
```

Comment:
maybe provide user feedback that the option is running
```diff
--- a/src/bench/bench.cpp
+++ b/src/bench/bench.cpp
@@ -58,6 +58,9 @@ void benchmark::BenchRunner::RunAll(const Args& args)
     std::smatch baseMatch;
 
     std::vector<ankerl::nanobench::Result> benchmarkResults;
+    if (args.one_iteration) {
+        std::cout << "Running with --one-iteration option, i.e. epochs(1).epochIterations(1)" << std::endl;
+    }
     for (const auto& p : benchmarks()) {
         if (!std::regex_match(p.first, baseMatch, reFilter)) {
```
output
```
$ time ./src/bench/bench_bitcoin -filter=WalletBalance*.* --one-iteration
Running with --one-iteration option, i.e. epochs(1).epochIterations(1)
Warning, results might be unstable:
* DEBUG defined
* CPU frequency scaling enabled: CPU 0 between 2,485.0 and 3,100.0 MHz
* Turbo is enabled, CPU frequency will fluctuate

Recommendations
* Make sure you compile for Release
* Use 'pyperf system tune' before benchmarking. See https://github.com/psf/pyperf

|               ns/op |                op/s |    err% |     total | benchmark
|--------------------:|--------------------:|--------:|----------:|:----------
```

Reply:
Revealing the fact that it's using `epochs(1).epochIterations(1)` seems like an unnecessary implementation detail for the `bench_bitcoin` user.

Reply:
I'd also prefer a config name like `--santiy-check`. Maybe in future this could do more to be quicker, like run all the benchmarks in parallel (I had a quick look if that works but got a segfault, probably due to some global states)

Reply:
> Revealing the fact that it's using epochs(1).epochIterations(1) seems like an unnecessary implementation detail for the bench_bitcoin user.

Yeah. That's too much detail. But adding a warning message that it's active to the output makes sense, I guess.

> I'd also prefer a config name like --sanity-check. Maybe in future this could do more to be quicker, like run all the benchmarks in parallel (I had a quick look if that works but got a segfault, probably due to some global states)

Agree on the rename. Running the benchmarks in parallel would be another stress test but it seems we're not ready for that :smile: 

Though ideally I would like to aim for <10ms per iteration for all benchmarks, so running 1 iteration in parallel shouldn't have that much effect. Running 1 of everything in serial is still supposed to be fast!

Reply:
> Agree on the rename.

@laanwj did you want to do that rename here? If not, I'd like to merge this soon.

Reply:
Yes, I'm going to make the requested changes here and re-push in a bit.